wordpressorg.test/provision/vvv-init.sh

```diff
@@ -19,6 +19,30 @@ WP_LOCALES=( ja es_ES )
 
 wme_svn_git_migration $SITE_DIR
 
+function wme_wporg_setup_cavalcade() {
+	# Cavalcade
+	printf "Installing Cavalcade..."
+	wme_noroot git clone https://github.com/humanmade/Cavalcade.git "${SITE_DIR}/wp-content/mu-plugins/cavalcade"
+	wme_noroot cp -f "${PROVISION_DIR}/cavalcade.php" "${SITE_DIR}/wp-content/mu-plugins/"
+
+	if [[ -d "/etc/cavalcade" ]]; then
+		pushd "/etc/cavalcade"
```

Comment:
TIL about `pushd`, thanks!

Reply:
I thought it was some newfangled thing when I saw it but apparently it's decades old and everywhere, even DOS! https://en.wikipedia.org/wiki/Pushd_and_popd